fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationOptionsDialog.kt

```diff
@@ -49,9 +52,45 @@ fun TranslationOptionsDialog(
 ) {
     TranslationOptionsDialogHeader(onBackClicked)
 
+    val translationOptionsListState = remember {
+        translationOptionsList.toMutableStateList()
+    }
+
+    EnabledTranslationOptionsItems(
+        translationOptionsListState,
+    )
+
     LazyColumn {
-        items(translationOptionsList) { item: TranslationSwitchItem ->
-            TranslationOptions(translationSwitchItem = item)
+        items(translationOptionsListState) { item: TranslationSwitchItem ->
+            val description =
+                stringResource(id = R.string.translation_option_bottom_sheet_switch_description)
```

Comment:
For the two in the language section, e.g.,  "Always translate Spanish" and "Never translate Spanish", could you use the description "Overrides offer to translate"? 

[String Figma](https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD/Full-Page-Translation?node-id=7202%3A122161&mode=dev) I'm basing this off of.